cp: add examples for common -L and Linux specific --parents (#4021) (#4026)
tldr-pages/tldr: pages/linux/cp.md

# cp

> Copy files and directories.

- Copy a file to another location:

`cp {{path/to/file.ext}} {{path/to/copy.ext}}`

- Copy a file into another directory, keeping the filename:

`cp {{path/to/file.ext}} {{path/to/target_parent_directory}}`

- Recursively copy a directory's contents to another location (if the destination exists, the directory is copied inside it):

`cp -r {{path/to/directory}} {{path/to/copy}}`

- Copy a directory recursively, in verbose mode (shows files as they are copied):

`cp -vr {{path/to/directory}} {{path/to/copy}}`

- Copy text files to another location, in interactive mode (prompts user before overwriting):

`cp -i {{*.txt}} {{path/to/target_directory}}`

> - Copy target pointed by symbolic link (instead just link):

Reviewer: This appears to be identical to the example in my previous comment, so the same applies here too.

`cp -L {{link}} {{path/to/copy}}`

> - Copy the source/path/to/file to path/to/copy, creating any missing intermediate directories from source path:

Reviewer: Perhaps this will read better?
```suggestion
- Use the full path of source files, creating any missing intermediate directories when copying:
```
